README.md

```diff
@@ -65,21 +51,38 @@ Installing
 
         $ ./ankisyncctl.py adduser <username>
 
-4. Run ankisyncd:
+4. Setup a proxy to unchunk the requests.
 
-        $ python -m ankisyncd
+    Webob does not support the header "Transfer-Encoding: chunked" used by Anki
+    and therefore ankisyncd sees chunked requests as empty. To solve this problem
+    setup Nginx (or any other webserver of your choice) and configure it to
+    "unchunk" the requests for ankisyncd.
 
----
+    For example, if you use Nginx  on the same machine as ankisyncd, you first
+    have to change the port in `ankisyncd.conf` to something other than `27701`.
+    Then configure Nginx to listen on port `27701` and forward the unchunked
+    requests to ankisyncd.
+
+    An example configuration with ankisyncd running on the same machine as Nginx
+    and listening on port `27702` may look like:
 
-<span id="readme-fn-01"></span>
-1. 2.1.9 is not supported due to [commit `95ccbfdd3679`][] introducing the
```

Comment:
Does this mean version 2.1.9 is not supported?

Reply:
I believe this restriction only applied for the server side version of Anki. With the changes introduced by this pull request we require the user to have the latest version of Anki available thorugh pip installed. 

Therefore this section of the Readme serves no purpose anymore.